src/InternetBankingServlet.java

```diff
@@ -0,0 +1,86 @@
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class InternetBankingServlet
+ */
+@WebServlet("/InternetBankingServlet")
+public class InternetBankingServlet extends HttpServlet 
+{
+	private static final long serialVersionUID = 1L;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public InternetBankingServlet() 
+    {
+        super();

```

Comment:
What does super() do? Add meaning full comments in every method/function/logic written.